src/platform/forms/tests/save-in-progress/01-sip-finish-later.cypress.spec.js

```diff
@@ -35,50 +34,43 @@ describe('SIP Finish Later', () => {
       .click();
 
     cy.url().should('not.contain', '/introduction');
-    cy.url().should('contain', '/veteran-information/birth-information');
+    cy.url().should('contain', '/first-page');
 
     cy.get('.schemaform-sip-save-link');
-    cy.get('#root_veteranSocialSecurityNumber').should(
-      'have.attr',
-      'value',
-      '123445544',
-    );
```

Comment:
Being this high up, this might be testing prefill functionality. Are we covering that somewhere else?

Reply:
I have updated the suite to add back in the tests that look for previously saved data to be prefilled and am using the email field to test